python/ray/tests/test_perf_integration.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+import pytest
+
+num_tasks_submitted = [10**n for n in range(0, 6)]
+num_tasks_ids = ["{}_tasks".format(i) for i in num_tasks_submitted]
+
+
+def init_ray(**kwargs):
```

Comment:
what's the point of the `init_ray` and `teardown_ray` functions?  Maybe just get rid of them.